templates/zombienet/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
```

Comment:
The templates are released on their own. This means that this template will not be part of the actual templates. (Not sure it would even be "released" somewhere right now, because this is probably a manual list somewhere)

This would also mean that you don't get this when downloading the `parachain-template` for example. If you wanted it this way, ignore my comment.

Reply:
Oks, in that case maybe is better to have the smoke test per template. I think also make more sense for end-users.

Reply:
Fwiw, I think it is less important to ship these tests when we run the process that mirrors them in their final repo. As long as they work here, we can be very certain that everything is fine and still works.

Reply:
Oks, so we keep this way (with zombienet test external to the templates itself). I will clean the code that was offloaded to zombienet-sdk. Thanks!